.github/workflows/.windows.yml

```diff
@@ -249,15 +272,20 @@ jobs:
     runs-on: ${{ inputs.os }}
     timeout-minutes: 120
     needs:
+      - test-prepare
       - build
       - integration-test-prepare
     strategy:
       fail-fast: false
       matrix:
+        storage: ${{ fromJson(needs.test-prepare.outputs.matrix) }}
         runtime:
           - builtin
           - containerd
         test: ${{ fromJson(needs.integration-test-prepare.outputs.matrix) }}
+        exclude:
+          - storage: snapshotter
+            runtime: builtin
```

Comment:
In the original PR, some jobs were dedup when running `builtin` along with `snapshotter`. This `exclude` makes sure we don't run them unnecessarily.

Reply:
snapshotter+builtin is an impossible scenario, since the builtin (fake) libcontainerd implementation doesn't implement snapshotters. Same thing is needed in `integration-test-report`.

Reply:
> Same thing is needed in `integration-test-report`.

:+1: